apps/uniforms-web/src/app/dynamic/form-editor/add-question-dialog/add-question-dialog.component.html

```diff
@@ -0,0 +1,64 @@
+<h2 mat-dialog-title align="center">
+  {{ baslik }} {{ data.isEditing ? 'Düzenle' : 'Ekle' }}
+</h2>
+<mat-dialog-content class="!p-2">
+  <div class="flex flex-col">
+    <uf-form-input
+      title="{{ this.data.type == 'title' ? 'Başlık' : 'Soru' }}"
+      [inputFormControl]="titleFC"
+    ></uf-form-input>
+    @if (this.data.type == 'dropdown') {
+      <uf-form-input class="flex-grow" title="Açılır Menü Yazı" [inputFormControl]="dropdownTextFC"></uf-form-input>
+    }
+    @if (!['title'].includes(this.data.type)){
```

Comment:
bunun yerine aşağıdaki gibi yazılabilir
```suggestion
    @if (this.data.type !== 'title') 
```